beacon_node/execution_layer/src/lib.rs

```diff
@@ -924,8 +925,21 @@ impl<T: EthSpec> ExecutionLayer<T> {
                     }
                 }
 
-                self.get_pow_block_hash_at_total_difficulty(engine, spec)
-                    .await
+                let block = self
+                    .get_pow_block_hash_at_total_difficulty(engine, spec)
```

Comment:
I think we should rename this function

```suggestion
                    .get_pow_block_at_total_difficulty(engine, spec)
```

Reply:
Nice catch!